### PR TITLE
Add constants and codes for early childhood

### PR DESCRIPTION
**Pull request by Mr0grog**
Vaccines for ages 6 months - 4 years (Pfizer) and 6 months - 5 years (Moderna) were approved today, and this adds constants for them as well as support for their known NDC and CVX codes so we are likely to be able to recognize them in the data.

Partially covers #710.

### CODE REVIEW

File: loader/src/smart-scheduling-links.js
Comment (astonm) [inline loader/src/smart-scheduling-links.js:46]: Any reason not to go ahead and rename this `pfizer_age_0_4`?

**Review by astonm — APPROVED** [review]
👍🏽

File: loader/src/smart-scheduling-links.js
Comment (Mr0grog) [reply loader/src/smart-scheduling-links.js:46]: Yep — NJ already knows about and is using it. I’d like to rename, but want a go-ahead from them first: #710.

Comment (Mr0grog) [reply loader/src/smart-scheduling-links.js:46]: Got a 👍 there so will also tweak that name before merging.

**Comment by Mr0grog** [comment]
OK! I renamed `pfizer_age_2_4` to `pfizer_age_0_4` and did a bunch of cleanup for all our fuzzy name matching to make it a little bit more strict and alert us when we see new kinds of pediatric vaccines. I also made it match the new 6 months - 4/5 years names that are currently showing up in Alaska’s PrepMod staging site (nothing in production for them yet).

Since I had to mess with the fuzzy matching, I also ran all the loaders to make sure they aren’t going to fail matching a vaccine that is currently working in production today.

**Review by astonm — APPROVED** [review]
Cool, lgtm. :shipit:

**Comment by Mr0grog** [comment]
Thanks!

**Pull request merged**
